src/device/insteonDeviceCommand/insteonDeviceCommand.ts

```diff
@@ -31,18 +31,57 @@ export = function(RED: Red){
 
 		/* Retrieve the config node */
 		this.deviceConfigNode = RED.nodes.getNode(props.device) as InsteonDeviceConfigNode;
+		
+		/* The command to be executed */
+		this.command = props.command;
+		
+		/* The onLevel to use (only valid for dimmers and when the command is 'on') */
+		this.onLevel = Math.round(parseInt(props.onLevel) / 100 * 255) as Byte;
+		
+		/* The onRamp is only used for the dimmers which support LightOnAtRate */
+		// this.onRamp = 0x99;
+		
+		/* If the device config node is already ready (meaning it's 'ready' event already fired before this subscribe node was created) */
+		if(this.deviceConfigNode.ready){
+			onReady(this,'Listening');
+		}else{
+			/* Wait and listen for the device to be ready */
+			this.deviceConfigNode.on('ready', p => onReady(this, p));		
+		}
 
-		/* Setting up PLM events */
-		this.deviceConfigNode.on('ready', p => onReady(this, p));
+		/* On input pass the message */
+		this.on('input', (msg) => onInput(msg, this));
 	});
 };
 
 //#region Event Functions
 
-function onReady(node: DeviceCommandNode, text: string){
+function onReady(node: DeviceCommandNode, text: string){	
 	node.log('Ready');
 
 	node.status({ fill: 'green', shape: 'dot', text });
 }
 
+function onInput(msg: any, node: DeviceCommandNode){
+	let device = node.deviceConfigNode?.device as any;
+	
+	let command = node.command === '' ? msg.payload : node.command;
```

Comment:
I think if there is a payload then that should overwrite the node settings.  That way you can have a node and override its behavior on the fly if needed

Reply:
I did it this way because of this [note](https://github.com/node-red/node-red/wiki/Deprecated:-Message-properties-overriding-set-node-properties) and also to make it consistent with some of the default Node RED nodes, such as the http-request node: 

![image](https://user-images.githubusercontent.com/4663918/72643938-c6d72c00-393d-11ea-9cbc-a4cd52d46f03.png)